README.md

```diff
@@ -50,26 +50,85 @@ Responsibilities and roles of the bridge:
 
 ## Usage
 
-### Install Dependencies
+There are two ways to deploy contracts:
+* install and use NodeJS
+* use Docker to deploy 
+
+### Deployment with NodeJS
+
+#### Install Dependencies
 ```bash
 npm install
 ```
-### Deploy
+#### Deploy
 Please the [README.md](deploy/README.md) in the `deploy` folder for instructions and .env file configuration
 
-### Test
+#### Test
 ```bash
 npm test
 ```
 
-### Flatten
+#### Flatten
+Fattened contracts can be used to verify the contract code in a block explorer like BlockScout or Etherscan.
+The following command will prepare flattened version of the contracts:
+
 ```bash
 npm run flatten
 ```
+The flattened contracts can be found in the `flats` directory.
+
+### Deployment in the Docker environment
+[Docker](https://www.docker.com/community-edition) and [Docker Compose](https://docs.docker.com/compose/install/) can be used to deploy contracts without NodeJS installed on the system. 
+If you are on Linux, we recommend you [create a docker group and add your user to it](https://docs.docker.com/install/linux/linux-postinstall/), so that you can use the CLI without `sudo`.
+
+#### Prepare the docker container
+```bash
+docker-compose up --build
+```
+_Note: The container must be rebuild every time when you change the code of contracts or the deployment scripts._
```

Comment:
_Note: The container must be rebuilt every time the code in a contract or deployment script is changed._